### PR TITLE
More user-friendly

### PR DESCRIPTION
**Pull request by Markandpreston1231**
### Related Issue/Addition to code

- Fixes #
- Addition to code.

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue) no
- [x] New feature (non-breaking change which adds functionality) kinda
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) no
- [ ] This change requires a documentation update no

### Proposed Changes
Just made the text more user-friendly

### Why is this change needed?
 
It's needed because...

### Additional Info
- any additional info for context

### Checklist:

- [x] My code follows the style guidelines of this project and I have read [CONTRIBUTING.md](/CONTRIBUTING.md) yes
- [x] I have performed a self-review of my own code yes
- [ ] I have commented my code, particularly in hard-to-understand areas no
- [ ] I have made corresponding changes to the documentation no
- [x] My changes generate no new warnings no
- [ ] Any dependent changes have been merged and published in downstream modules no
- [x] I have checked my code and corrected any misspellings yes

---

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -198,7 +198,7 @@ update () {
     # In case requirements are updated
 
     echo "--------------------------"
-    echo "Updated!"
+    echo "Updated sucsessfuly!"
```
Comment (Firecul) [inline install.sh:201]:
```suggestion
    echo "Updated successfully!"
```

File: install.sh [hunk 2]
```diff
@@ -169,7 +169,7 @@ install_MAIN () {
 
     # Done!
 
-    printf "Dependencies and Program installed into .\YT-Spammer-Purge!\nNow follow these instructions to get a client_secrets.json file!\nhttps://github.com/ThioJoe/YT-Spammer-Purge/wiki/Instructions:-Obtaining-an-API-Key\n\nYou may run this script again inside your installation to update.\n"
+    printf "YT Spammer Purge has been installed succsesfully! Its in .\YT-Spammer-Purge.\nNow follow these instructions to get a client_secrets.json file!\nhttps://github.com/ThioJoe/YT-Spammer-Purge/wiki/Instructions:-Obtaining-an-API-Key\n\nYou may run this script again inside your installation to update.\n"
```
Comment (Firecul) [inline install.sh:172]:
```suggestion
    printf "YT Spammer Purge has been installed successfully! It's in .\YT-Spammer-Purge.\nNow follow these instructions to get a client_secrets.json file!\nhttps://github.com/ThioJoe/YT-Spammer-Purge/wiki/Instructions:-Obtaining-an-API-Key\n\nYou may run this script again inside your installation to update.\n"
```

File: install.sh [hunk 3]
```diff
@@ -105,7 +105,7 @@ install_macos() {
 
 install_python_requirements () {
     python3 -m pip install -q -r requirements.txt --user && \
-        echo "Python requirements installed." || exit 1
+        echo "Requirements for Python installed." || exit 1
```
Comment (Rohith0009) [inline install.sh:108]:
```suggestion
        echo "Requirements for Python are installed." || exit 1
```

File: install.sh [hunk 4]
```diff
@@ -183,7 +183,7 @@ update () {
     # This assumes we are in the YT-Spammer-Purge directory
     check_python_requirements
     clear
-    echo "We will now attempt to update YT-Spammer-Purge."
+    echo "Will now check for updates."
```
Comment (Rohith0009) [inline install.sh:186]:
```suggestion
    echo "Checking for updates"
```

**Review by Rohith0009 — COMMENTED** [review]
These Changes Would be good

File: install.sh (hunk 4 above)
Comment (Markandpreston1231) [reply install.sh:186]: - [ ] I will now check the code

**Comment by Markandpreston1231** [comment]
I am going to review and apply the changes

File: install.sh (hunk 4 above)
Comment (Markandpreston1231) [reply install.sh:186]: I will not do this as the script will tell the user if they want to continue so it wouldn't make sense.

**Comment by A802526** [comment]
FB_IMG_1668504920490

**Comment by Markandpreston1231** [comment]
So what will we do

**Pull request closed without merging**